src/sagas/trending.js

```diff
@@ -0,0 +1,13 @@
+import { takeLatest, call, put } from 'redux-saga/effects';
+import { LOAD_TRENDING_REQUEST, loadTrendingSuccess } from '../actions/trending';
+import { fetchTrending } from '../api';
+
+export function* fetchTrendingSaga() {
+  const response = yield call(fetchTrending);
```

Comment:
How do you know if this call effect failed? And how would you reflect it on the UI?

Reply:
I did not add error handling for errors due to lack of time 😅 But probably would refactor to something like:

```js
try {
  const response = yield call(fetchTrending);
  yield put(loadTrendingSuccess(response));
} catch (e) {
  // this would put an erro flag in the trending reducer to update the UI accordingly
  yield put(loadTrendingError(e));
}
```

Reply:
Nice, that'll do. I noticed you did mention in your pending items that error handling was in fact pending. I missed that 😇 .